unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuildModeHUD/Scripts/Common/EntityInformationController.cs

```diff
@@ -0,0 +1,265 @@
+
+        }
+        else
+        {
+            entityInformationView.SetSmartItemListViewActive(false);
+        }
+
+        entityInformationView.SetEntityThumbnailEnable(false);
+        CatalogItem entitySceneObject = entity.GetCatalogItemAssociated();
+        GetThumbnail(entitySceneObject);
+        UpdateLimitsInformation(entitySceneObject);
+        UpdateEntityName(entityInformationView.currentEntity);
+        UpdateInfo(entityInformationView.currentEntity);
+    }
+
+    internal void GetThumbnail(CatalogItem catalogItem)
+    {
+        if (catalogItem == null)
+            return;
+
+        var url = catalogItem.thumbnailURL;
+
+        if (string.IsNullOrEmpty(url))
+            return;
+
+        string newLoadedThumbnailURL = url;
+        var newLoadedThumbnailPromise = new AssetPromise_Texture(url);
+        newLoadedThumbnailPromise.OnSuccessEvent += SetThumbnail;
+        newLoadedThumbnailPromise.OnFailEvent += x => { Debug.Log($"Error downloading: {url}"); };
+        AssetPromiseKeeper_Texture.i.Keep(newLoadedThumbnailPromise);
+        AssetPromiseKeeper_Texture.i.Forget(loadedThumbnailPromise);
+        loadedThumbnailPromise = newLoadedThumbnailPromise;
+    }
+
+    internal void SetThumbnail(Asset_Texture texture)
+    {
+        entityInformationView.SetEntityThumbnailEnable(true);
+        entityInformationView.SetEntityThumbnailTexture(texture.texture);
+    }
+
+    internal void UpdateEntityName(DCLBuilderInWorldEntity entity)
+    {
+        if (entity == null)
+            return;
+
+        string currentName = entity.GetDescriptiveName();
+        entityInformationView.SeTitleText(currentName);
+
+        if (!isChangingName)
+            entityInformationView.SetNameIFText(currentName);
+    }
+
+    internal void UpdateLimitsInformation(CatalogItem catalogItem)
+    {
+        if (catalogItem == null)
+        {
+            entityInformationView.SeEntityLimitsLeftText("");
+            entityInformationView.SeEntityLimitsRightText("");
+            return;
+        }
+
+        string leftText = $"ENTITIES: {catalogItem.metrics.entities}\n" +
+                          $"BODIES: {catalogItem.metrics.bodies}\n" +
+                          $"TRIS: {catalogItem.metrics.triangles}";
+
+        string rightText = $"TEXTURES: {catalogItem.metrics.textures}\n" +
+                           $"MATERIALS: {catalogItem.metrics.materials}\n" +
+                           $"GEOMETRIES: {catalogItem.metrics.meshes}";
+
+        entityInformationView.SeEntityLimitsLeftText(leftText);
+        entityInformationView.SeEntityLimitsRightText(rightText);
+    }
+
+    public void Enable()
+    {
+        entityInformationView.SetActive(true);
+        entityInformationView.isEnable = true;
+    }
+
+    public void Disable()
+    {
+        entityInformationView.SetActive(false);
+        entityInformationView.isEnable = false;
```

Comment:
Same as stated Pato with `SetActive `and `isEnable`

Reply:
As we talked, I have removed the `isEnable` property as it was not needed. Changes done in https://github.com/decentraland/explorer/pull/2103/commits/1d121c8015bcea7e49a2aa9ce8b78ed09394d716